scripts/build-glean-metadata.js

```diff
@@ -1,47 +1,44 @@
-const fetch = require("node-fetch");
```

Comment:
Should probably just delete this file in a separate PR -- it's a half-baked attempt at creating a search index.